src/osdc/Objecter.cc

```diff
@@ -2339,6 +2342,10 @@ void Objecter::_send_op_account(Op *op)
     case CEPH_OSD_OP_CMPXATTR: code = l_osdc_osdop_cmpxattr; break;
     case CEPH_OSD_OP_RMXATTR: code = l_osdc_osdop_rmxattr; break;
     case CEPH_OSD_OP_RESETXATTRS: code = l_osdc_osdop_resetxattrs; break;
+    case CEPG_OSD_OP_TMAPUP: code = l_osdc_osdop_tmap_up; break;
```

Comment:
Typos

Reply:
... this is still an issue causing a compile failure:

> /home/jenkins-build/build/workspace/ceph-pull-requests/src/osdc/Objecter.cc: In member function 'void Objecter::_send_op_account(Objecter::Op*)':
> /home/jenkins-build/build/workspace/ceph-pull-requests/src/osdc/Objecter.cc:2344:10: error: 'CEPG_OSD_OP_TMAPUP' was not declared in this scope
>      case CEPG_OSD_OP_TMAPUP: code = l_osdc_osdop_tmap_up; break;
>           ^~~~~~~~~~~~~~~~~~
> /home/jenkins-build/build/workspace/ceph-pull-requests/src/osdc/Objecter.cc:2344:10: note: suggested alternative: 'CEPH_OSD_OP_TMAPUP'
>      case CEPG_OSD_OP_TMAPUP: code = l_osdc_osdop_tmap_up; break;
>           ^~~~~~~~~~~~~~~~~~
>           CEPH_OSD_OP_TMAPUP
> src/osdc/CMakeFiles/osdc.dir/build.make:88: recipe for target 'src/osdc/CMakeFiles/osdc.dir/Objecter.cc.o' failed
> make[3]: *** [src/osdc/CMakeFiles/osdc.dir/Objecter.cc.o] Error 1

Reply:
Sorry,It was my negligence.The file has been modified.